README.md

```diff
@@ -0,0 +1,17 @@
+# Judge Daemon 4 for VJ4
+
+Next generation of judge daemon for [Vijos](https://vijos.org), built with asyncio on Python 3.5.
+
+## Prerequisites
+
+* [Linux Kernel 4.0+]
```

Comment:
Never tried with 4.0. We should stick to 4.4.

Could be written as Linux 4.4+ to be cool as linux refers to the linux kernel